exercise/staff_views.py

```diff
@@ -65,7 +69,45 @@ def get_common_objects(self):
         self.all = self.request.GET.get('all', None)
         self.all_url = self.exercise.get_submission_list_url() + "?all=yes"
         self.submissions = qs if self.all else qs[:self.default_limit]
-        self.note("all", "all_url", "submissions", "default_limit")
+        
+        # Regrade button text changes dynamically if regrade is ongoing.
```

Comment:
To me it seems like the cleanest inheritance path would be ListSub***View --> (new) ExerciseListBaseView --> ExerciseBaseView, and overriding get_common_objects(), because the regrade functionality also needs definitions from ExerciseBaseView. Would go with that, unless there are other proposals. Better proposals for the new class name are also welcome.